presto-main/src/main/java/com/facebook/presto/dispatcher/DispatchManager.java

```diff
@@ -89,6 +92,25 @@
 
     private final SecurityConfig securityConfig;
 
+    /**
+     * Instantiates a new Dispatch Manager.
+     * This is a critical component in resource management section of the query.
+     * This is for the pre-queuing part of central manager for queries
+     *
+     * @param queryIdGenerator the query id generator
```

Comment:
It would be helpful to add bit more description about these param. I am not sure if expanding the variable name is sufficient.

Reply:
fixed